### PR TITLE
Modified the config.get() function to behave as a singleton and insur…

### PR DESCRIPTION
**Pull request by AaronH88**
…e the config is only read once.

# Description
Also updated the kafka mod to the correct one.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in Ibutsu:
https://url.corp.redhat.com/ibutsu-runs-d2b866a3-06fb-4892-a4aa-c1c4d2272c04

File: go.mod [hunk 1]
```diff
@@ -25,7 +25,6 @@ require (
 	github.com/segmentio/kafka-go v0.4.32
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/viper v1.12.0
-	gopkg.in/confluentinc/confluent-kafka-go.v1 v1.8.2
```
Comment (ldjebran) [inline go.mod:28]: is this removed by accident ?

Comment (ldjebran) [reply go.mod:28]: Got it we had 2 conflicting versions in dependency

Comment (AaronH88) [reply go.mod:28]: No, 
Confluent is confusing as it tells us two different ways to pull their library in Go. 
Their docs asks us to use `gopkg.in/confluentinc/confluent-kafka` and this gives us version 1.8.2
Their github asks us to use `github.com/confluentinc/confluent-kafka` and this gives us version 1.9.0 

As github seems to be more us to date and more stable, I decided to use this one :smile:

**Review by ldjebran — APPROVED** [review]
ACK

**Comment by ldjebran** [comment]
@mergifyio requeue

**Pull request merged**